dev-util/sysprof/sysprof-48.0.ebuild

```diff
@@ -0,0 +1,111 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org gnome2-utils greadme meson systemd xdg
+
+DESCRIPTION="System-wide Linux Profiler"
+HOMEPAGE="http://sysprof.com/"
+
+LICENSE="GPL-3+ GPL-2+"
+API_VERSION="4"
+SLOT="0/${API_VERSION}"
+KEYWORDS="~amd64 ~arm64 ~loong ~x86"
+IUSE="gtk systemd test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=dev-libs/glib-2.76.0:2
+	gtk? (
+		>=gui-libs/gtk-4.15:4
+		>=gui-libs/libadwaita-1.6_alpha:1
```

Comment:
It seems that these version requirements also changed:
-  dependency('libadwaita-1', version: '>= 1.6.alpha'),
-  dependency('libpanel-1', version: '>= 1.7.0'),
+  dependency('libadwaita-1', version: '>= 1.6.0'),
+  dependency('libpanel-1', version: '>= 1.4'),